doc/book/src/installation.md

```diff
@@ -35,24 +39,24 @@ cargo binstall maelstrom-run
 
 These commands retrieve the pre-built binaries from the [Maelstrom GitHub
 release page](https://github.com/maelstrom-software/maelstrom/releases). If you
-don't have `cargo-binstall`, you can just manually install the binaries from the
-releases page. For example:
+don't have `cargo-binstall`, you can directly install the pre-built binaries by
+simply untarring the release artifacts. For example:
 
 ```bash
 wget -q -O - https://github.com/maelstrom-software/maelstrom/releases/latest/download/cargo-maelstrom-x86_64-unknown-linux-gnu.tgz | tar xzf -
 ```
 
-This will download and extract the latest release of `cargo-maelstrom` for x86 Linux.
+This will download and extract the latest release of `cargo-maelstrom` for Linux on the x86-64 architecture.
 
 ## Installing Using Nix
 
-We have a `nix.flake` file, so you can install all Maelstrom binaries with something like:
```

Comment:
Generally, I found the use of "we" a little off-putting. Substituting the project name generally read better to me.

Reply:
Okay, I updated it.